cmd/zoekt-mirror-gerrit/main.go

```diff
@@ -73,9 +73,40 @@ func newLoggingClient() *http.Client {
 	}
 }
 
+type RepoFormatEnum string
+
+const (
+	WithServerHost    RepoFormatEnum = "WithServerHost"
+	WithoutServerHost RepoFormatEnum = "WithoutServerHost"
+)
+
+// Implementing flag.Value interface
+type RepoFormatEnumValue struct {
+	value *RepoFormatEnum
+}
+
+func (l *RepoFormatEnumValue) String() string {
+	return string(*l.value)
+}
+
+func (l *RepoFormatEnumValue) Set(s string) error {
+	switch s {
+	case string(WithServerHost):
+		*l.value = WithServerHost
+		return nil
+	case string(WithoutServerHost):
+		*l.value = WithoutServerHost
+		return nil
+	}
+	return fmt.Errorf("invalid RepoFormatEnum '%s'", s)
+}
+
 func main() {
 	dest := flag.String("dest", "", "destination directory")
 	namePattern := flag.String("name", "", "only clone repos whose name matches the regexp.")
+	// Default value for
+	repoFormat := WithServerHost
+	flag.Var(&RepoFormatEnumValue{&repoFormat}, "repo-name-format", "the format of the local repo name in zoekt (WithServerHost or WithoutServerHost)")
```

Comment:
minor: the CamelCase values read a bit weird for a command line argument. Maybe better values for this would be `qualified` and `project`?

Reply:
Done